com.qq.QQ.yaml

```diff
@@ -102,7 +102,19 @@ modules:
         commands:
           - export ZYPAK_LD_PRELOAD="/app/etc/preload/rename.so"
           - export TMPDIR="$XDG_RUNTIME_DIR/app/$FLATPAK_ID"
-          - exec zypak-wrapper /app/extra/QQ/qq "$@"
+          - |
+            FLAGS='--enable-features=UseSkiaRenderer,WaylandWindowDecorations'
+
+            if [[ -e "$XDG_RUNTIME_DIR/${WAYLAND_DISPLAY:=wayland-0}" ]]
+            then
+                FLAGS="$FLAGS --ozone-platform-hint=auto --enable-wayland-ime"
+            fi
+
+            if [[ $XDG_SESSION_TYPE == "wayland" ]] && [ -c /dev/nvidia0 ]
```

Comment:
Is this really needed? I mean that if we are using hybrid GPU system, this may results that NVIDIA GPU is used by accident and more power is used.

Reply:
I am using hybird GPU system, this line is used to disable gpu sandbox if system is running on Nvidia GPU since Nvidia device is still not very friendly to Wayland. Although Discord doesn't have this now, other Electron based apps may still have this check. But I am also not very sure if this check is really necessary now.

Reply:
I finally just keep the minimal flags to launch the app on Wayland.